src/aiperf/endpoints/huggingface_generate.py

```
@@ -0,0 +1,125 @@
+        self, response: InferenceServerResponse
+    ) -> ParsedResponse | None:
+        """Handle token stream (SSE or chunked) responses."""
+        chunks = []
+        try:
+            for packet in response.packets:
+                # Each packet is an SSEField(name='data', value='...')
+                if packet.name != "data":
+                    continue
+                try:
+                    json_obj = json.loads(packet.value)
+                    if "token" in json_obj and json_obj["token"]:
```

Ruff (0.14.2)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)